mbbsd/mail.c

```diff
@@ -1227,6 +1253,8 @@ mail_account_sysop(void)
     char            save_title[STRLEN];
     char            tmp_title[STRLEN-20];
```

Comment:
I forgot why STRLEN-20?
1. why not STRLEN? I am nervous to read that and afraid STRLEN<20. 
2. what is "20"?